swagger/v1/component_schemas/ECFParticipantRetainedDeclaration.yml

```diff
@@ -1,5 +1,6 @@
 description: "An ECF retained participant declaration"
 type: object
+additionalProperties: false
```

Comment:
i tried setting these to `additional_properties` but does not output `additionalProperties` which is the correct openapi key